lib/iris/tests/test_coding_standards.py

```diff
@@ -219,11 +219,10 @@ def last_change_by_fname():
             msg = "{} is not a git repository."
             raise ValueError(msg.format(IRIS_REPO_DIRPATH))
 
-        # Call "git whatchanged" to get the details of all the files and when
+        # Call "git log" to get the details of all the files and when
         # they were last changed.
-        # TODO: whatchanged is deprecated, find an alternative Git command.
         output = subprocess.check_output(
-            ["git", "whatchanged", "--pretty=TIME:%ct", "--i-still-use-this"],
+            ["git", "log", "--name-status", "--pretty=TIME:%ct"],
```

Comment:
@melissaKG Awesome! Thanks for this 💯 

Note that, `git log --raw --pretty=TIME:%ct` gives a closer output similar to `git whatchanged --pretty=TIME:%ct`, but as far as I'm concerned the information not provided by the truncated `--name-status` also works here, so let's roll with this change 👍